src/EditorFeatures/Test/Diagnostics/DiagnosticDataTests.cs

```diff
@@ -46,7 +46,7 @@ public async Task DiagnosticData_GetText2()
             var code = @"
 ";
 
-            await VerifyTextSpanAsync(code, -1, 30, 40, 40, new TextSpan(0, code.Length));
+            await VerifyTextSpanAsync(code, 0, 30, 40, 40, new TextSpan(code.Length, 0));
```

Comment:
VerifyTextSpanAsync was updated to directly construct LinePos structs.  Those will throw on negative values so these tests were updated.  This isn't an issue in practice as we go through teh AdjustBoudaries helper in real code which clamps these values to be leagal.  looking to see if we can unify the test/product code here.